### PR TITLE
docs: alter database ttl

### PR DESCRIPTION
**Pull request by CookiePieWw**
## What's Changed in this PR

#1313 update docs for alter database ttl.

Waiting for https://github.com/GreptimeTeam/greptimedb/pull/5035

## Checklist

- [x] Please confirm that all corresponding versions of the documents have been revised.
- [x] Please ensure that the content in `sidebars.ts` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

**Review by nicecui — COMMENTED** [review]
please modify the document outline to:
```
# Alter

`ALTER` can be used to modify any database options, table options or metadata of the table, including:
...

## Alter database
### Syntax
### Examples

## Alter table
### Syntax
### Examples
```

File: docs/reference/sql/alter.md [hunk 1]
```diff
@@ -1,13 +1,54 @@
...
+`ALTER` can be used to modify any database options, table options or metadata of the table, including:
 
+* Modify database options
 * Add/Drop/Modify a column
 * Rename a table
+* Modify table options
 
-## Syntax
+## ALTER DATABASE
+
```
Comment (nicecui) [inline docs/reference/sql/alter.md:11]:
```suggestion

`ALTER DATABASE` statements can be used to modify the options of databases.

```

File: docs/reference/sql/alter.md [hunk 2]
```diff
@@ -1,13 +1,54 @@
...
+### Syntax
+
+```sql
+ALTER DATABASE db
+   [SET <option_name>=<option_value> [, ...]
+    | UNSET <option_name> [, ...]
+   ]
+```
+
```
Comment (nicecui) [inline docs/reference/sql/alter.md:20]:
```suggestion

Currently following options are supported:
- `ttl`: the default retention time of data in database.

```

File: docs/reference/sql/alter.md [hunk 3]
```diff
@@ -1,13 +1,54 @@
...
+### Examples
+
+#### Modify database options
+
+`ALTER DATABASE` statements can be used to modify the options of databases.
+
+Currently following options are supported:
+- `ttl`: the default retention time of data in database.
+
```
Comment (nicecui) [inline docs/reference/sql/alter.md:29]:
```suggestion
```

File: docs/reference/sql/alter.md [hunk 4]
```diff
@@ -1,13 +1,54 @@
...
+```sql
+ALTER DATABASE db
+   [SET <option_name>=<option_value> [, ...]
+    | UNSET <option_name> [, ...]
+   ]
+```
+
+### Examples
+
+#### Modify database options
```
Comment (nicecui) [inline docs/reference/sql/alter.md:23]:
```suggestion
#### Modify retention time of data
```

**Review by nicecui — COMMENTED** [review]
please also update the zh doc according to the en comments, thanks

File: docs/reference/sql/alter.md (hunk 1 above)
Comment (CookiePieWw) [reply docs/reference/sql/alter.md:11]: Not sure if it's the place. If there're more `ALTER DATABASE` stmt like rename database then we need to change it again.

**Review by nicecui — APPROVED** [review]

**Review by fengjiachun — APPROVED** [review]
LGTM

**Pull request merged**